rsa/prime.py

```diff
@@ -171,6 +171,86 @@ def getprime(nbits):
 
             # Retry if not prime
 
+small_primes=(3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61,
+              67, 71, 73, 79, 83, 89, 97)
+def prime_sieve(start,end):
```

Comment:
The name suggest the function yields primes, but this isn't the case. Probably it's better to call it `composite_sieve`.

Also include in the docstring that this is a generator.